Sources/FluentBenchmark/Tests/TimestampTests.swift

```diff
@@ -50,6 +50,9 @@ extension FluentBenchmarker {
                     XCTFail("Timestamp decoding from database output failed with error: \(error)")
                 }
             }).wait()
+
+            try event.delete(on: self.database).wait()
+            try XCTAssertEqual(Event.query(on: self.database).all().wait().count, 0)
```

Comment:
It doesn't look like `Event` has an `on: .delete` timestamp so I'm not sure this test does anything.